metron-deployment/roles/ambari_config/tasks/main.yml

```diff
@@ -26,16 +26,15 @@
   retries: 5
   delay: 10
 
-- name : check if ambari-server is up on {{ ambari_host }}:{{ambari_port}}
+- name : Wait for Ambari to start; http://{{ ambari_host }}:{{ ambari_port }}
   wait_for :
     host: "{{ ambari_host }}"
     port: "{{ ambari_port }}"
-    delay: 120
-    timeout: 300
+    timeout: 600
```

Comment:
There is no need to always wait 2 minutes for Ambari to be ready.  Most often it is already up and kicking by the time we get here.  Rather than have a forced delay, I just added the delay duration to the overall timeout parameter in case there is a delay in getting Ambari going.